src/import/engine.rs

```diff
@@ -254,6 +256,13 @@ impl SSTWriter {
         })
     }
 
+    fn tune_config(env: Arc<Env>, cf_opts: &mut ColumnFamilyOptions) {
+        use DBCompressionType::Lz4;
+        cf_opts.set_env(env);
+        cf_opts.compression_per_level(&[Lz4, Lz4, Lz4, Lz4, Lz4, Lz4, Lz4]);
+        cf_opts.compaction_priority(CompactionPriority::OldestLargestSeqFirst);
```

Comment:
What consideration makes you use this priority?

Reply:
Sorry, forgot to delete this line.